project-base/storefront/components/Blocks/Popup/ManageCustomerUserPopup.tsx

```diff
@@ -0,0 +1,248 @@
+                firstName: customerUserManageProfileFormData.firstName,
+                lastName: customerUserManageProfileFormData.lastName,
+                telephone: customerUserManageProfileFormData.telephone,
+                roleGroupUuid: customerUserManageProfileFormData.roleGroup.value,
+            },
+        });
+
+        handleUpdateResult(
+            addUserResult.data?.AddNewCustomerUser !== undefined,
+            addUserResult.error,
+            formProviderMethods,
+            formMeta.messages,
+        );
+
+        updatePortalContent(null);
+    };
+
+    const handleUpdateResult = (
+        isResultOk: boolean,
+        error: CombinedError | undefined,
+        formProviderMethods: UseFormReturn<CustomerUserManageProfileFormType>,
+        messages: { success?: string; error?: string },
+        callbacks?: { success?: () => void; error?: () => void },
+    ) => {
+        if (isResultOk) {
+            if (messages.success !== undefined) {
+                showSuccessMessage(messages.success);
+            }
+            if (callbacks?.success !== undefined) {
+                callbacks.success();
+            }
+        }
+
+        if (error === undefined) {
+            return;
+        }
+
+        const { userError, applicationError } = getUserFriendlyErrors(error, t);
+
+        if (applicationError !== undefined) {
+            if (messages.error !== undefined) {
+                showErrorMessage(messages.error, GtmMessageOriginType.other);
+            }
+            if (callbacks?.error !== undefined) {
+                callbacks.error();
+            }
+        }
+
+        if (userError?.validation !== undefined) {
+            for (const fieldName in userError.validation) {
+                formProviderMethods.setError(
+                    fieldName as Path<CustomerUserManageProfileFormType>,
+                    userError.validation[fieldName],
+                );
+            }
+        }
+    };
+
+    return (
+        <Popup className="w-11/12 lg:w-4/5 vl:w-auto overflow-x-auto">
+            <FormProvider {...formProviderMethods}>
+                <Form onSubmit={formProviderMethods.handleSubmit(onSubmitCustomerUserManageProfileFormHandler)}>
+                    <FormContentWrapper>
+                        <FormBlockWrapper>
+                            <FormHeading>{t('Personal data')}</FormHeading>
+                            <TextInputControlled
+                                control={formProviderMethods.control}
+                                formName={formMeta.formName}
+                                name={formMeta.fields.email.name}
+                                render={(textInput) => <FormLine bottomGap>{textInput}</FormLine>}
+                                textInputProps={{
+                                    label: formMeta.fields.email.label,
+                                    required: false,
+                                    disabled: mode === 'edit',
```

Comment:
Could you please double-check this condition? The input field is disabled even when I'm trying to add a new user.

Reply:
Fixed.